src/Nethermind/Nethermind.Blockchain/SynchronizationManager.cs

```diff
@@ -234,6 +239,7 @@ public async Task AddPeer(ISynchronizationPeer synchronizationPeer)
 
             var peerInfo = new PeerInfo(synchronizationPeer);
             _peers.TryAdd(synchronizationPeer.NodeId, peerInfo);
+            _transactionPool.AddPeer(peerInfo.Peer);
```

Comment:
should it not be a role of PeerManager to pass peers both to sync manager and tx pool?

Reply:
Ok, I can move the code to PeerManager, it seemed easier, for now, to keep it inside SyncManager.

Reply:
I understand, however I already have a feeling that sync manager is overloaded with responsibilities

Reply:
Will check if it can be easily moved to PeerManager.